src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let cost = 0;
+
+  if (days < 3) {
+    cost = days * 40;
+  } else {
+    if (days >= 3 && days < 7) {
+      cost = days * 40 - 20;
+    } else {
+      if (days >= 7) {
+        cost = days * 40 - 50;
+      }
+    }
+  }
```

Comment:
Good job!
But lets make some variables for values, because for now we don't understand what it is - 40, 50, 20 ...